components/clientComponents/forms/FormattedDate/FormattedDate.tsx

```diff
@@ -184,16 +191,15 @@ export const FormattedDate = (props: FormattedDateProps): React.ReactElement =>
               <label className="!mr-2 mb-2" htmlFor={`${name}-${part}`}>
                 {t(`formattedDate.${part}`)}
               </label>
+              <div id={`${id}-description-day`} hidden>
+                {t("number")}
+              </div>
               <input
                 name={`${name}-${part}`}
                 id={`${name}-${part}`}
-                type="number"
-                min={1}
-                max={
```

Comment:
Won't removing this cause issues?

Reply:
With a type=text the min/max doesn't really make sense. And even on a type=number the min/max only impact when increasing/decreasing a number using the stepper controls

Reply:
Let's make sure to get Dave's input on this part.